### PR TITLE
MARP-77: add adobe-esign-connector

### PR DESCRIPTION
**Pull request by linhpd-axonivy**

### CODE REVIEW

**Review by linhpd-axonivy — COMMENTED** [review]
Hi @ntqdinh-axonivy ,
Please help me update logo.png and meta.json

**Comment by linhpd-axonivy** [comment]
Hi @ivy-sgi , @andreasbalsiger ,
We are preparing to publish the Adobe E-Sign connector to Marketplace. Please review the product name, description, tags, and other metadata to ensure they accurately convey the connector's value proposition and capabilities.
Thank you!

**Comment by linhpd-axonivy** [comment]
Hi @andreasbalsiger , @ivy-sgi ,
Please help me review the product name, description, tags, and other metadata to ensure they accurately convey the connector's value proposition and capabilities.
Thank you!

File: market/connector/adobe-esign-connector/meta.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://json-schema.axonivy.com/market/10.0.0/meta.json",
+    "id": "adobe-esign-connector",
+    "name": "Adobe Esign Connector",
```
Comment (andreasbalsiger) [inline market/connector/adobe-esign-connector/meta.json:4]: based on [Wikipedia](https://en.wikipedia.org/wiki/Adobe_Sign), the official name is "Adobe Acrobat Sign", please change the name to "Adobe Acrobat Sign"

File: market/connector/adobe-esign-connector/meta.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "description": "Use this connector to create, sign document with Adobe Sign.",
```
Comment (andreasbalsiger) [inline market/connector/adobe-esign-connector/meta.json:5]: Please reword to:
Use this connector to send, sign, track, and manage signature processes in Axon Ivy

File: market/connector/adobe-esign-connector/meta.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    "platformReview": "4.5",
+    "sourceUrl": "https://github.com/axonivy-market/adobe-esign-connector",
+    "statusBadgeUrl": "https://github.com/axonivy-market/adobe-esign-connector/actions/workflows/ci.yml/badge.svg",
+    "language": "English",
+    "industry": "Cross-Industry",
+    "tags": [ "e-signature" ],
+    "mavenArtifacts": [
+        {
+            "repoUrl": "https://maven.axonivy.com",
+            "name": "Adobe Esign Connector",
```
Comment (andreasbalsiger) [inline market/connector/adobe-esign-connector/meta.json:16]: Do you need to change all these names to the official name, see line 4 above. I dont like to have different names afterwards and "Adobe Acrobat Sign" is supposed to be the right one.

**Review by andreasbalsiger — COMMENTED** [review]
Andreas' comments

File: market/connector/adobe-esign-connector/meta.json (hunk 1 above)
Comment (ntqdinh-axonivy) [reply market/connector/adobe-esign-connector/meta.json:4]: Hi @andreasbalsiger,
Do we need to change the name of repo, package and other to "Adobe Acrobat Sign" also?

File: market/connector/adobe-esign-connector/meta.json (hunk 3 above)
Comment (andreasbalsiger) [reply market/connector/adobe-esign-connector/meta.json:16]: I am puristic so I would say yes... but I am not aware if this causes additional problems we dont want. Maybe @ivy-sgi, you can share your opinion here?

File: market/connector/adobe-esign-connector/meta.json (hunk 1 above)
Comment (andreasbalsiger) [reply market/connector/adobe-esign-connector/meta.json:4]: I am puristic so I would say yes... but I am not aware if this causes additional problems we dont want. Maybe @ivy-sgi , you can share your opinion here?

Comment (ivy-sgi) [reply market/connector/adobe-esign-connector/meta.json:4]: Hi @andreasbalsiger ! Changing the repo name is a bit dangerous and we are not 100 sure if we know all dependencies. How worse would it be for you to approve this PR as it is?

Comment (andreasbalsiger) [reply market/connector/adobe-esign-connector/meta.json:4]: Hi @ivy-sgi : Fair enough... lets approve it like this!

**Pull request merged**